lib/src/modules/apm.dart

```diff
@@ -54,6 +61,13 @@ class APM {
   /// [String] id of the trace.
   /// [String] key of attribute.
   /// [String] value of attribute.
+  ///
+  /// Deprecated: from version v12.7.2.
```

Comment:
This should be `v13.0.0` now. Also, why do we need this line in the docs comment when we are using `@Deprecated`?
(Same for other instances of this).